From c2b1fdce7a530c5141e1d251e74a01bfbd0e2e53 Mon Sep 17 00:00:00 2001
From: wu guoxing <b39297@freescale.com>
Date: Wed, 16 Nov 2011 11:56:45 +0800
Subject: ENGR00162325 mx6:Make GPU as kernel module defaultly

GC355 needs to map all reserved memory and consume too much vmalloc space.
So it needs a special kernel command line vmalloc=xxxM.
otherwise, kernel can not boot

Signed-off-by: Wu Guoxing <b39297@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 0bd025f..621b76f 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux/arm 2.6.38 Kernel Configuration
-# Fri Nov 11 09:57:21 2011
+# Wed Nov 16 10:57:43 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -1979,7 +1979,7 @@ CONFIG_MXC_ASRC=y
 #
 # MXC GPU support
 #
-CONFIG_MXC_GPU_VIV=y
+CONFIG_MXC_GPU_VIV=m
 
 #
 # ANATOP_THERMAL
-- 
1.7.9.5

